lib/src/firebase_auth_mocks_base.dart

```diff
@@ -81,6 +94,14 @@ class MockFirebaseAuth implements FirebaseAuth {
     return Future.value(userCredential);
   }
 
+  Future<UserCredential> _fakeSignUp({bool isAnonymous = false}) {
+    final userCredential = MockUserCredential(isAnonymous, mockUser: _mockUser);
+    _currentUser = userCredential.user;
+    stateChangedStreamController.add(_currentUser);
+    userChangedStreamController.add(_currentUser);
+    return Future.value(userCredential);
```

Comment:
Since this code is exactly the same as _fakeSignIn, I'll merge your PR, then replace this code by a call to _fakeSignIn.

Reply:
https://github.com/atn832/firebase_auth_mocks/commit/8cc38edbccd2b0e91926d77d1fa5dd11089c42ee